openapi/giphy/client.bal

```diff
@@ -0,0 +1,187 @@
+import  ballerina/http;
```

Comment:
Missing license header

Reply:
fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/fcd3580d6892ae0cfbfa1e9449cc814999a11f4b